.github/workflows/preview.yml

```diff
@@ -0,0 +1,45 @@
+name: Preview
+on:
+  pull_request:
+    branches: 
+      - master
+      - develop
+      - 'release/**'
+jobs:
+  deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+      
+      - name: install deps
+        run: yarn
+      
+      - name: test
+        run: ./node_modules/lerna/cli.js run test
```

Comment:
This is OK for now, but ideally we'll just limit this to a subset of our tests. Calling `lerna run test` will trigger the `test` script from each of our packages.

Once we improve our testing story, we can revisit this!